Userland/Applications/Spreadsheet/ExportDialog.cpp

```diff
@@ -198,22 +196,15 @@ ErrorOr<void> ExportDialog::make_and_run_for(StringView mime, Core::File& file,
             array.append(sheet.to_json());
 
         auto file_content = array.to_deprecated_string();
-        bool result = file.write(file_content);
-        if (!result) {
-            return Error::from_string_literal("Unable to save file.");
-        }
-
-        return {};
+        return file->write_entire_buffer(file_content.bytes());
     };
 
     if (mime == "text/csv") {
         return export_xsv();
     } else if (mime == "application/x-sheets+json") {
         return export_worksheet();
     } else {
-        auto page = GUI::WizardPage::construct(
-            "Export File Format",
-            DeprecatedString::formatted("Select the format you wish to export to '{}' as", LexicalPath::basename(file.filename())));
+        auto page = GUI::WizardPage::construct("Export File Format", "Select the format you wish to export file as");
```

Comment:
That looks to me like a UI regression. I know that Core::Stream::File doesn't provide a file name, could you pass it separately to the function?